crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -277,13 +276,8 @@ impl SpaceViewClass for TimeSeriesSpaceView {
 
         let scalar_axis =
             ViewProperty::from_archetype::<ScalarAxis>(blueprint_db, ctx.blueprint_query, view_id);
-        let mut y_range = scalar_axis.component_or_fallback::<Range1D>(ctx, self, state)?;
-        if y_range.start() >= y_range.end() {
-            // Ensure that the range is valid - egui_plot might debug_assert otherwise.
-            // `next_up_f64` should be sufficient, but empirically it's not always enough
-            // (likely we're loosing precision due to some calculations down the line)
-            *y_range.end_mut() = y_range.start() + 1.0;
-        }
+        let y_range = scalar_axis.component_or_fallback::<Range1D>(ctx, self, state)?;
+        let y_range = make_range_sane(y_range);
```

Comment:
nice. We have to do the same on the bar chart though I believe!

Reply:
There is no `Range1D` in `crates/re_space_view_bar_chart/src/space_view_class.rs`

Reply:
but @jleibs opened a pr specifically about the bar chart breaking under zero range